docs/getting-started/install.md

```diff
@@ -26,16 +26,116 @@ Athena's Javascript should be added before the closing <code>&lt;/body&gt;</code
 
 **package.json**
 <pre><code>{
-  "athena-framework": "github:UCF/Athena-Framework"
-}</code></pre>
\ No newline at end of file
+  &hellip;
+  "dependencies": {
+    "jquery": "^3.2.1",
+    "tether": "^1.4.0"
+    "athena-framework": "github:UCF/Athena-Framework"
+  }
+}</code></pre>
+
+### Bower
+If you are using the <a href="https://bower.io/">Bower</a> package manager, you can add Athena as a depency by running `bower install athena-framework --save` or by adding Athena to you `bower.json` file and running `bower install`. As with NPM, ensure you also include jQuery and Tether.
+
+**bower.json**
+<pre><code>{
+  &hellip;
+  "dependencies": {
+    "jquery": ">=1.9.1",
+    "tether": "^1.4.0",
+    "git://github.com/UCF/Athena-Framework": "v1.0.0-alpha"
+  }
+}</code></pre>
+
+___
+
+## Adding to Existing Project Using Bootstrap
+
+Athena is a complete replacement for Bootstrap. If you are already using Bootstrap on an existing project and want to use Athena instead, you should remove Bootstrap’s CSS and Javascript completely. Athena is designed to run as a completely standalone framework.
+
+<pre><code><strike>&lt;link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"&gt;</strike>
+<span class="highlight">&lt;link rel="stylesheet" href="/css/framework.min.css"&gt;</span>
+
+&hellip;
+
+&lt;script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.slim.min.js"&gt;&lt;/script&gt;
+&lt;script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"&gt;&lt;/script&gt;
+<strike>&lt;script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"&gt;</strike>
+<span class="highlight">&lt;script type="text/javascript" src="/js/framework.min.js"&gt;&lt;/script&gt;</span>
+</code></pre>
+
+___
+
+## Webfonts &amp; Configuration
+
+Athena is built to utilize premium fonts provided by the <a href="http://www.typography.com/cloud/welcome/">Cloud.typography service</a> by default, with webfont fallbacks (available in the `/src/fonts/` directory). If neither Cloud.Typography or webfont fallbacks are utilized, Athena will default to use appropriate system fonts.
+
+### Included Fonts
+
+To utilize fallback webfonts, copy the `/fonts/` directory (from the `/src/fonts/` in the project files) to the same `/src/` folder you added the CSS and JS files. The CSS is built to include webfonts from that directory.
+
+<pre><code> ├── css/
+<strong> ├── fonts/
+ │   ├── tulia/
+ │   ├── ucf-condensed-alt/
+ │   └── ucf-sans-serif-alt/</strong>
+ └── js/
+</code></pre>
+
+### Cloud.Typography Premium Font Configuration
+
+The sans-serif font family used in Athena is **Gotham**. Gotham can be installed in your project via the <a href="http://www.typography.com/cloud/welcome/">Cloud.typography service.</a>
```

Comment:
Maybe update this first line to something like `While Athena provides fallback webfonts out-of-the-box, the preferred font families for Athena are only available via the <a href="http://www.typography.com/cloud/welcome/">Cloud.typography service</a>. Athena users are responsible for creating their own Cloud.typography accounts and maintaining their subscriptions to the service.`